rootfs/etc/nginx/lua/balancer/sticky.lua

```diff
@@ -59,31 +67,82 @@ local function set_cookie(self, value)
   end
 end
 
+function _M.get_last_failure()
```

Comment:
Does this have to be public function?

Reply:
This function is made "public" to change its behavior for request failure simulation (see [this](https://github.com/fedunineyu/ingress-nginx/blob/change-upstream-on-error-with-sticky-session/rootfs/etc/nginx/lua/test/balancer/sticky_test.lua#L243) test).